docs/features.md

```diff
@@ -48,16 +48,16 @@ If a `pulse` is not specified, then a disruption will not be pulsing.
 
 ## Targeting
 
-**NEW:** StaticTargeting currently defaults to true. It will default to false after some transition time. [Read StaticTargeting](#StaticTargeting-(current-default-behaviour)).
+**NEW:** StaticTargeting currently defaults to false. Please mention explicitely if you wish to activate it. [Read StaticTargeting](#StaticTargeting-(current-default-behaviour)).
 
 The `Disruption` resource uses [label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) to target pods and nodes. The controller will retrieve all pods or nodes matching the given label selector and will randomly select a number (defined in the `count` field) of matching targets. It's possible to specify multiple label selectors, in which case the controller will select from targets that match all of them. Once applied, you can see the targeted pods/nodes by describing the `Disruption` resource.
 
 **NOTE:** If you are targeting pods, the disruption must be created in the same namespace as the targeted pods.
 
-### StaticTargeting (current default behaviour)
+### StaticTargeting
 
 When `StaticTargeting` is activated, targeting is limited to a single target selection at the disruption's creation. It allows for more controlled disruption impact and propagation, as the targets will never change and _can_ be compensated for in case they are made useless. Its major limit is not being able to follow targets through deployments/rollouts.
-By setting the `StaticTargeting` flag to `false` in the [Disruption's yaml spec](../examples/static_targeting.yaml), you activate a constant re-targeting for this disruption. This means at any given time, any target within the selector's scope will be added to the target list and be disrupted.
+By setting the `StaticTargeting` flag to `false` -or empty- in the [Disruption's yaml spec](../examples/static_targeting.yaml), there is a constant re-targeting for this disruption. This means at any given time, any target within the selector's scope will be added to the target list and be disrupted.
 This is a feature to use with care, as it can quickly get out of control: per example, a disruption targeting 100% of an application's pod will affect all existing **and** future pods which can appear once the disruption started. As long as this 100% disruption exists, there will be no spared pod.
```

Comment:
Given this is our default now, this is a weird way to phrase this?

Reply:
pushed a rephrasing, should be better now !